src/tlo/methods/breast_cancer.py

```diff
@@ -646,6 +647,7 @@ def __init__(self, module, person_id):
         self.TREATMENT_ID = "BreastCancer_Investigation"
         self.EXPECTED_APPT_FOOTPRINT = self.make_appt_footprint({"Over5OPD": 1, "Mammography": 1})
         self.ACCEPTED_FACILITY_LEVEL = '3'  # Mammography only available at level 3 and above.
+        # TODO: what this means, should be the mammography done within this event, or the biopsy, or both?
```

Comment:
Not sure about this. Better ask Andrew.

Reply:
@EvaJanouskova I don't know for sure but I think we should consider this as just biopsy.  As you saw in the guidelines, population screening for mammography is not done.  So I think the mammography can be removed here.

Reply:
I've update the comment from `# Mammography only available at level 3 and above.` to `# Biopsy only available at level 3 and above.`, is that correct?